src/tbbbind/tbb_bind.cpp

```diff
@@ -266,7 +300,17 @@ class platform_topology {
     void fill_constraints_affinity_mask(affinity_mask input_mask, int numa_node_index, int core_type_index, int max_threads_per_core) {
         __TBB_ASSERT(is_topology_parsed(), "Trying to get access to uninitialized platform_topology");
         __TBB_ASSERT(numa_node_index < (int)numa_affinity_masks_list.size(), "Wrong NUMA node id");
+#if __TBB_PREVIEW_TASK_ARENA_CONSTRAINTS_EXTENSION_PRESENT
```

Comment:
`__TBB_PREVIEW_TASK_ARENA_CONSTRAINTS_EXTENSION_PRESENT` is not defined in this file. 
I suppose you should add `__TBBBIND_BUILD` in https://github.com/uxlfoundation/oneTBB/blob/dev/dnmokhov/2021.2/include/oneapi/tbb/detail/_config.h#L361.

Reply:
Thanks for pointing this out, @sunxiaoxia2022! Added.